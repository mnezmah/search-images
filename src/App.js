import React from 'react';
import { Provider } from 'react-redux'
import store from './store'
import FormContainer from './components/Form/FormContainer'
import TileContainer from './components/Tile/TileContainer'

import './App.css';

function App() {
  return (
   <Provider store={store}>
      <div className="App">
        <FormContainer />
        <TileContainer />
      </div>
   </Provider>
  );
}

export default App;
