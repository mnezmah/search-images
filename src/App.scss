@import "./Styling/main";

body {
  background-color: $main-background;
}

.App {
  text-align: center;
  margin: 0 auto;

  @media (min-width: 1152px) and (max-width: 1279.5px) {
    max-width: 90%;
  }

  @media (min-width: 1280px) {
    max-width: 87%;
  }
}
