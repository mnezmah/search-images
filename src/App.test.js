import React from 'react'
import { shallow } from 'enzyme'
import App from './App';

test('should render App corectly', () => {
  const wrapper = shallow(<App />)
  expect(wrapper).toMatchSnapshot()
})