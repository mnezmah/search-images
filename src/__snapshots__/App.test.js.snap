// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App corectly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <div
    className="App"
  >
    <Connect(FormContainer) />
    <Connect(TileContainer) />
  </div>
</Provider>
`;
